templates/base.html

```diff
@@ -76,12 +76,15 @@
 				<h3><span class="glyphicon glyphicon-random"></span>&nbsp;&nbsp;&nbsp;Module Relations</h3>
 				<ul>
 					<li><a id="sidebar" href="/overlappingModules">Overlapping Modules</a></li>
-					<li><a id="sidebar" href="/moduleTakenPriorToOthers">Modules Taken Prior to Other Modules</a></li>
-					<li><a id="sidebar" href="/nonOverlappingModules">Non-Overlapping Modules</a></li>
-				    <!--ul>
-					   <li><a href="#">Before Specified Module</a></li>
-				        <li><a href="#">Before Internship</a></li>
-				    </ul-->
+					<li><a href="javascript:void(0)">Prior Modules Taken</a></li>
```

Comment:
Sidebar will be redesigned at one of the refinement sprints. Please raise an issue in Github for this UI issue.